paasta_tools/oom_logger.py

```diff
@@ -209,18 +228,32 @@ def send_sfx_event(service, instance, cluster):
         counter.count()
 
 
+def get_containerd_container(
+    is_cri_containerd: bool, container_id: str
+) -> containers_pb2.Container:
+    with grpc.insecure_channel("unix:///run/containerd/containerd.sock") as channel:
+        containersv1 = containers_pb2_grpc.ContainersStub(channel)
+        if is_cri_containerd:
+            namespace = "k8s.io"
+        else:
+            namespace = "moby"
```

Comment:
hmm, could we use the containerd library for both cases? if so, we could remove the conditional in `main()` :)

otherwise, we can just assume that if this method is being called we want the k8s namespace

Reply:
yea, I was thinking of also just assuming if we call this function then we are using containerd but I wasn't sure if we  would still need the moby ns